main/main.cpp

```diff
@@ -423,7 +423,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --doctool [<path>]                Dump the engine API reference to the given <path> (defaults to current dir) in XML format, merging if existing files are found.\n");
 	OS::get_singleton()->print("  --no-docbase                      Disallow dumping the base types (used with --doctool).\n");
 	OS::get_singleton()->print("  --build-solutions                 Build the scripting solutions (e.g. for C# projects). Implies --editor and requires a valid project to edit.\n");
-	OS::get_singleton()->print("  --dump-gdextension-interface      Generate GDExtension header file 'gdnative_interface.h' in the current folder. This file is the base file required to implement a GDExtension.\n");
+	OS::get_singleton()->print("  --dump-gdextension-interface      Generate GDExtension header file 'gdextension_interface.h' in the current folder. This file is the base file required to implement a GDExtension.\n");
 	OS::get_singleton()->print("  --dump-extension-api              Generate JSON dump of the Godot API for GDExtension bindings named 'extension_api.json' in the current folder.\n");
```

Comment:
I wonder if we should rename this one to `--dump-gdextension-api` to match `--dump-gdextension-interface`... but then should the JSON be `gdextension_api.json` :thinking:

:canned_food: :worm: 

Scope creep anyway for this PR ;)

Reply:
You tell me. But I think I agree with the fact it could go in another PR :grin: .
I think we could discuss that in the GDExtension meeting, I'll add it as a topic to discuss.